### DISCUSSION TITLE
What is the difference between Reference “Tutorial” section and Guide?

### DISCUSSION THREAD

**Discussion by awxiaoxian2020** [mdn/mdn-community#639] [Content]
The tutorial in Reference and the content in Guide seem to be the same design goal.

**Comment by yin1999** [comment]
Hey @awxiaoxian2020. This may relate to this project: openwebdocs/project#185. Most of the docs under guide group have been moved. I believe that after the remaining work is completed, we could avoid causing such confusion to readers :)

**Reply by awxiaoxian2020** [reply]
I found I understand something wrong. I will close this discussion and open a new one.
